src/arch/riscv/isa/decoder.isa

```diff
@@ -2240,9 +2240,13 @@ decode QUADRANT default Unknown::unknown() {
                 }}, OPFVV, VectorDummyOp);
                 // VWFUNARY0
                 0x10: decode VS1 {
-                    0x00: VectorNonSplitFormat::vfmv_f_s({{
-                        Fd_bits = Vs1_vu[0];
-                    }}, OPFVV, VectorDummyOp);
+                    0x00: decode VM {
+                        // The encodings corresponding to the masked versions
+                        // (vm=0) of vfmv.f.s are reserved
+                        0x1: VectorNonSplitFormat::vfmv_f_s({{
+                            Fd_bits = Vs1_vu[0];
```

Comment:
Unrelated to this change but still a bug - this should be `Vs2`.

Reply:
I agree with you.

Reply:
> Unrelated to this change but still a bug - this should be `Vs2`.

I agree with you.

Reply:
I have a fix here:
https://github.com/rivosinc/plct-gem5/commit/a4cfc00006a39c7c450591054f2bfbbe5b812850

We can leave this code as is and I'll include the fix in my next PR after this one is merged.